tsconfig.json

```diff
@@ -4,11 +4,16 @@
     "emitDeclarationOnly": false,
     "incremental": false,
     // Required to run benchmark command from root directory
-    "typeRoots": ["node_modules/@types", "./types"],
+    "typeRoots": [
+      "node_modules/@types",
+      "./types",
+      "${configDir}/node_modules/@types",
+      "${configDir}/types"
```

Comment:
We can use template variables in configs now. 

https://devblogs.microsoft.com/typescript/announcing-typescript-5-5/#the-${configdir}-template-variable-for-configuration-files